mleap-runtime/src/main/scala/ml/combust/mleap/runtime/transformer/Transformer.scala

```diff
@@ -33,6 +34,8 @@ trait Transformer extends AutoCloseable {
     */
   def transform[TB <: TransformBuilder[TB]](builder: TB): Try[TB]
 
+  def getSchema(): Try[Seq[StructField]]
```

Comment:
I've updated the method on Transformer to be getFields() and the method in MleapService which returns the Try[StructType] to be getSchema()